backend/transcribee_backend/models/task.py

```diff
@@ -64,6 +65,24 @@ class Task(TaskBase, table=True):
     state: TaskState = TaskState.NEW
     state_changed_at: datetime.datetime = Field(default_factory=now_tz_aware)
 
+    initial_cost: (float | None) = None
+
+    def guess_initial_cost(self, media_duration_sec: float) -> float | None:
```

Comment:
Why not calculate this as-needed and make this a property function?
```suggestion
    @property
    def initial_cost(self, media_duration_sec: float) -> float | None:
```

Reply:
We can do that, but then we have to query the document each time

Reply:
done